lib/asn1.js

```diff
@@ -206,13 +208,17 @@ asn1.create = function(tagClass, type, constructed, value) {
     value = tmp;
   }
 
-  return {
+  var obj = {
     tagClass: tagClass,
     type: type,
     constructed: constructed,
     composed: constructed || forge.util.isArray(value),
     value: value
   };
+  if(options && 'raw' in options) {
+    obj.raw = options.raw;
```

Comment:
I'm not a huge fan of the name `raw`. I don't feel like it's descriptive enough for what's going on, but since I don't have a better name, we can go with it. It's not difficult to alias later and deprecate.